app/util/decorators.py

```diff
@@ -6,7 +6,7 @@
 from app.util.secret import Secret
 
 
-def retry_on_exception(exceptions, num_attempts=10, retry_delay=0.1):
+def retry_on_exception_linear(exceptions, num_attempts=10, retry_delay=0.1):
```

Comment:
Now I think this could just be:
```python
def retry_on_exception_linear(exceptions, num_attempts=10, retry_delay=0.1):
    return retry_on_exception_exponential_backoff(exceptions, initial_delay=0.1, total_delay=num_attempts * retry_delay, exponential_factor=1):
```

Or I guess we could just delete it!